tests/autologging/test_training_session.py

```diff
@@ -17,23 +17,28 @@ class Grandchild:
     pass
 
 
+parent = Parent()
+child = Child()
+grand_child = Grandchild()
```

Comment:
```suggestion
PARENT = Parent()
CHILD = Child()
GRAND_CHILD = Grandchild()
```

Can we make these variables uppercase?